packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/Characteristics/components/FilterButton.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from 'enzyme'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import zooTheme from '@zooniverse/grommet-theme'
+import { CloseButton, Media } from '@zooniverse/react-components'
+
+import FilterButton, { StyledFilter } from './FilterButton'
+
+describe('Component > FilterButton', function () {
+  let wrapper, onFilterSpy
+
+  before(function () {
+    onFilterSpy = sinon.spy()
+    wrapper = mount(
+      <FilterButton
+        characteristicId='LK'
+        checked={false}
+        onFilter={onFilterSpy}
+        valueImageSrc='bird-icon.svg'
+        valueLabel='bird'
+      />, {
+        wrappingComponent: Grommet,
+        wrappingComponentProps: { theme: zooTheme }
+      }
```

Comment:
```suggestion
      />
```
These tests work without the wrapping component.